.github/workflows/bisects.yml

```diff
@@ -5,21 +5,22 @@ on:
     types: created
 
 jobs:
-  test:
-    runs-on: ubuntu-latest
+  bisects:
+    if: |
+      github.event_name == 'issue_comment' && startsWith(github.event.comment.body, '!nim ') && github.event.issue.pull_request == null && github.event.comment.author_association != 'NONE'
```

Comment:
Explanation:

`github.event_name == 'issue_comment'` ➡️ Only run for Issues.

`startsWith(github.event.comment.body, '!nim ')` ➡️ Comment must start with "!nim "

`github.event.issue.pull_request == null` ➡️ Issue is not a Pull Request 
(for some weird reason pull requests are also issues on github API).